Readme.md

```diff
@@ -198,6 +201,12 @@ This event is triggered if `console.log` is used on the page. But this event is
 ##### .on('page-alert', message)
 This event is triggered if `alert` is used on the page.
 
+##### .on('page-prompt', message [, response])
+This event is triggered if `prompt` is used on the page.  Note that unless the `preload` script is overridden, the prompt will cause an exception to be thrown.
```

Comment:
Can this say:
Note that unless `window.prompt` is overridden in a preload script, ...

(and similarly below for page-confirm)